packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -187,11 +187,11 @@ const Alert = ({
   const _title = title ? (
     <BaseBox marginBottom="spacing.2">
```

Comment:
maybe add `textAlign="left"` here or the outermost element so you don't have to add on individual items?

Reply:
Done.